yang-packages/draft-ietf-netmod-yang-packages.txt

```diff
@@ -1361,55 +1276,43 @@ Internet-Draft                YANG Packages                 October 2022
             one of the included packages.";
        }
 
-       leaf local {
```

Comment:
I recall we discussed this but not sure whether we closed on whether we can do without it.

Reply:
Yes, I don't think that this is a closed issue.  I was discussing it with Jason, and maybe deleting it is a bit pre-emptive.  I'm sure that there will be some further discussion on whether this is a good flag to have.